go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/ThreeDotsLabs/watermill-amqp
 
-go 1.17
```

Comment:
I don't think you should remove this.

Reply:
good catch @sagikazarmark I force pushed to undo.  Had to resolve a few mod issues on the CLI must have removed as part of that.  I just force pushed with the go 1.17 back in there thanks.